.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,44 +1,46 @@
-## Reference
+## Description for [TASK_ID](TASK_URL)
 
-[TASK_ID](TASK_URL)
+DESCRIPTION_HERE
 
-## Overview
-
-Brief description of what this PR does, and why it is needed.
-
-### Acceptance Criteria
-
-- [ ] CRITERION_HERE
-
-### Demo
+### 🖼 Demo Images
 
 <table>
 <tr><th>Before</th><th>After</th></tr>
 <tr>
 <td>
 
-![](BEFORE_IMAGE_URL_HERE)
+![](https://github.com/cremalab/<REPO_NAME>/blob/<BASE_BRANCH>/.loki/reference/chrome_<COMPONENT_NAME>_<STORY_NAME>.png?raw=true)
 
 </td>
 <td>
 
-![](AFTER_IMAGE_URL_HERE)
+![](https://github.com/cremalab/<REPO_NAME>/blob/<FULL_COMMIT_SHA>/.loki/reference/chrome_<COMPONENT_NAME>_<STORY_NAME>.png?raw=true)
 
 </td>
 </tr>
 </table>
 
-## Reviewers' Checklist (go ahead, check 'em)
+> Test images work nicely for this (table is optional)
 
-- [ ] Merge destination is correct
-- [ ] Tests cases have been added where appropriate (Jest, Loki, Cypress)
-- [ ] Code is correct as understood and conforms to quality standards
-- [ ] Acceptance criteria have been met
+### 📋 Acceptance Criteria
 
-## Testing Instructions
+- [ ] CRITERION_HERE
+
+> Checked off by the PR assignee(s)
```

Comment:
@roblafeve One question, when you say the 'PR assignee(s)' that is a little confusing to me because I thought that 'reviewers' checked off acceptance criteria. This could just be my misunderstanding of who checks off what - but just wanted to double check.

Reply:
Yup the confusion is understandable because that's what we said it meant yesterday 😄. I attempted to think of it from the perspective of the reviewer—I would hope/assume that the "Assignee" has evaluated and confirmed that they have met the Acceptance Criteria. So because we still have a Review Checklist item that is "Acceptance Criteria have been met" I felt that it made this feel a little more like a collaborative effort between Assignee and Reviewer 🤷‍♂ Totally open to adjusting this 👍

Reply:
I think it makes sense that the assignee would go through the Acceptance Criteria as a last check of "ok did I complete all of these tasks?" when creating the Pull Request. I feel like I've seen you guys do that anyway when you've been working on tickets - so it makes sense to just check them off as part of the process.